test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildServerIntegrationTest.cs

```diff
@@ -195,5 +223,13 @@ public async Task Build_WithWhiteSpaceInPipeName_BuildsSuccessfully()
                 fixture.Dispose();
             }
         }
+
+        private class TestProjectDirectory : ProjectDirectory
```

Comment:
Why do we need this?

Reply:
Without this I couldn't construct `ProjectDirectory` which is used in [line 159](https://github.com/aspnet/Razor/pull/2687/files#diff-134e5ebcd6abe67765dbe638b558c9fcR159)